src/com/amazon/ion/benchmark/IonSchemaUtilities.java

```diff
@@ -166,6 +168,77 @@ public static String parseTextConstraints(IonStruct constraintStruct, String key
         return constraint;
     }
 
+    /**
+     * This method is used for checking whether the constraint 'valid_value' has the annotation 'range'.
+     * If 'valid_value' has the annotation 'range', this method will return the range value in IonList format, else it will return null.
+     * @param constraintStruct is the Ion struct which contain the current constraint field.
+     * @return the result of checking whether constraint 'valid_value' contains the annotation 'range'.
+     * @throws Exception if error occurs when building the IonReader.
+     */
+    public static IonList checkValidRange(IonStruct constraintStruct) throws Exception {
```

Comment:
The method name here doesn't seem to identify with what it is doing. Maybe a name like `getValidValuesAsRange()` would work.

Reply:
> The method name here doesn't seem to identify with what it is doing. Maybe a name like `getValidValuesAsRange()` would work.

Sounds good.